src/views/TipCommentsView.vue

```diff
@@ -145,12 +145,9 @@ export default {
   methods: {
     async sendTipComment() {
       if (this.USE_DEEP_LINKS) {
-        const url = new URL(`${process.env.VUE_APP_WALLET_URL}/comment`);
-        url.searchParams.set('id', this.tip.id);
-        url.searchParams.set('text', this.newComment);
-        url.searchParams.set('x-success', window.location);
-        url.searchParams.set('x-cancel', window.location);
-        window.location = url;
+        window.location = createDeepLinkUrl(
+          { type: 'comment', id: this.tip.id, text: this.newComment },
+        );
```

Comment:
```suggestion
        window.location = createDeepLinkUrl(
          { type: 'comment', id: this.tip.id, text: this.newComment, }
        );
```
or better

```suggestion
        window.location = createDeepLinkUrl({
          type: 'comment',
          id: this.tip.id,
          text: this.newComment,
        });
```

Reply:
Why? There is no second parameter IMHO this is less right than some formatting
I guess this is some bug in the es-linter when formatting this (weird) way